Src/ILGPU/Interop.cs

```diff
@@ -70,27 +70,28 @@ public static int SizeOf<T>(T structure)
         /// <typeparam name="TSecond">
         /// The base type that should be represented with <typeparamref name="TFirst"/>.
         /// </typeparam>
+        /// <param name="numSecondElements">The number of <typeparamref name="TSecond"/> elements to be stored.</param>
         /// <returns>
-        /// The number of required <typeparamref name="TFirst"/> instances to store an
-        /// instance of type <typeparamref name="TSecond"/>.
+        /// The number of required <typeparamref name="TFirst"/> instances to store <paramref name="numSecondElements"/>
+        /// instances of type <typeparamref name="TSecond"/>.
         /// </returns>
         [SuppressMessage(
            "Microsoft.Design",
             "CA1004:GenericMethodsShouldProvideTypeParameter",
             Justification = "The type is required for the computation of the " +
             "field offset")]
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        public static int ComputeRelativeSizeOf<TFirst, TSecond>()
+        public static int ComputeRelativeSizeOf<TFirst, TSecond>(int numSecondElements = 1)
```

Comment:
I am not sure if the global program analysis accepts optional parameters without warnings...

Reply:
Split into two methods - the original without parameters (marked as `Obsolete`), and a new method with required parameter.